tests/app/service/test_rest.py

```diff
@@ -1059,6 +1059,28 @@ def test_get_all_notifications_for_service_including_ones_made_by_jobs(
         assert response.status_code == 200
 
 
+def test_get_only_api_created_notifications_for_service(
+    notify_api,
+    notify_db,
+    notify_db_session,
+    sample_service
+):
+    with notify_api.test_request_context(), notify_api.test_client() as client:
```

Comment:
i'm going to keep pushing my `client` pytest fixture until everyone uses it!

just swap out notify_api for client in the parameters and then you don't need this line